### ISSUE TITLE
Uncaught TypeError: t.seconds.toNumber is not a function

### ISSUE THREAD

**Issue by ethanrublee** [stephenh/ts-proto#108]
I'm running into what I think is a bug in ts-proto when decoding a message from binary:
Generated code for message containing a google.protobuf.Timestamp field:
```
function fromTimestamp(t: Timestamp): Date {
  let millis = t.seconds.toNumber() * 1_000;
  millis += t.nanos / 1_000_000;
  return new Date(millis);
}
```

Error:
```
Uncaught TypeError: t.seconds.toNumber is not a function
```

**Comment by ethanrublee** [comment]
Replacing the generated line with the following fixes this issue:

```
  let millis = Number(t.seconds) * 1_000;
```

**Comment by ethanrublee** [comment]
This is also resolved by using the protoc option ``--ts_proto_opt=forceLong=string``

Issue occurs when using the ``--ts_proto_opt=forceLong=long`` flag.

It looks like this is related to using webpack (I'm getting this error in the browser) - https://github.com/stephenh/ts-proto/issues/78

The suggested work around fixes the TypeError and I can continue to use forceLong=long in protoc.

I placed the following as the first import statements at the top of index.tsx
```js
import * as protobuf from "protobufjs/minimal";
import * as Long from "long";
protobuf.util.Long = Long;
protobuf.configure();
```

**Comment by stephenh** [comment]
Hey, I'm trying to write a test to reproduce this in the browser, but not having much luck (I'm historically an 80/20 backend/frontend engineer, so this is my first time futzing with karma). Here's the PR:

https://github.com/stephenh/ts-proto/pull/111

If you have any ideas on how to make that work (to eventually reproduce this behavior when would mean getting this bug fixed :-), that'd be great. Thanks!

**Comment by stephenh** [comment]
@ethanrublee do you think this is a dup of #78 ? I.e. that one is about automatically adding the `protobuf.util.Long = Long` to any output that uses `forceLong`, which afaiu would fix this for you as well right?

**Comment by ethanrublee** [comment]
It does seem like a dup of #78.  One way to reproduce this is to create a message with a google protobuf Timestamp field.  I'm not sure how to be helpful on the automated testing side... however I get these errors reliably when using ts-proto in a simple react application, at runtime in chrome, when decoding a message with a timestamp field from binary data.

**Comment by stephenh** [comment]
Closing as a dup of #78, which is fixed in the latest 1.33.0 release. (@ethanrublee let me know if that's not the case, thanks!)
